tests/ui.rs

```diff
@@ -223,14 +227,15 @@ fn ui(
     with_dependencies: Dependencies,
     tmpdir: &Path,
 ) -> Result<()> {
-    let msg = format!("## Running ui tests in {path} against miri for {target}");
+    let msg = format!("## Running ui tests in {path} for {target}");
```

Comment:
The "against miri" part is a leftover from ancient times when we also ran these tests in rustc.